CHANGELOG.md

```diff
@@ -16,6 +16,7 @@
 - `re update source` can now update the source's transform tag
 - `re get source` and `re get sources` will show bucket name if exists.
 - `re get comments` can now download predictions within a given timerange
+- `re create quota` to set a quota in a tenant
```

Comment:
We discussed whether this should be `create` or `update` given that in some sense there's always a tenant quota, even if it's just the default. But since we're going to add a `delete` method and we don't want to create completely new subcommands, calling this `create` for symmetry seems best.